### DISCUSSION TITLE
Question for both induction and inhibition existing

### DISCUSSION THREAD

**Discussion by miyayayayaa** [Open-Systems-Pharmacology/Forum#1382] [Questions & Problems]
Dear all,
       I have a molecule that is metabolized only by CYP3A4, and is a CYP3A4 inducer, and has TDI. But there is a problem with the model results. At the same dose, simulation time, and population, if only CYP3A4 induction parameters (Emax=19.4 EC50=0.78umol/L) is set, the AUC  is 81.09umol/L; if CYP3A4 induction and inhibition parameters (Emax=19.4 EC50=0.78umol/L Ki=1.9umol/L)  are set at the same time, the AUC is 80.69umol/L. This phenomenon cannot be explained principlely. So dose anyone meet this question or any explanation? 

Thanks,
Min

**Comment by StephanSchaller** [comment]
Dear @miyayayayaa , users will only be able to help if you can describe your issue in a more quantitative way, e.g.: what is the expected outcome, do you have data? How does the simulation compare to data? etc...

**Comment by Yuri05** [comment]
moving to the forum
